### PR TITLE
Add infos about Onboarding and Cluster access policies

### LINKED ISSUE #71: Reference the Mila IT Onboarding in Getting Started
Side note: I  wonder if we should reference the Mila IT Onboarding instead of  `get your Mila cluster account ready`. This would add a link to the Intranet, ~~which is not accessible from the Web~~, but this could be less ambiguous for newcomers.

```diff
- **Before starting:** get your Mila cluster account ready. Typically, your
+ **Before starting:** get your Mila cluster account ready by completing [the Onboaring cluster training](https://sites.google.com/mila.quebec/mila-intranet/it-infrastructure/it-onboarding-training#h.kq4eovdw12dk). Typically, your
```

@tesfaldet @lavoiems @vict0rsch @fosterrath-mila any thoughts?

_Originally posted by @btravouillon in https://github.com/mila-iqia/mila-docs/pull/70#r706862794_

### PR DESCRIPTION
**Pull request by gandarioult**
resolves #71

Rajout du lien vers le IT Onboarding Guide + Politique d'accès au Cluster (Core + Staff)

### CODE REVIEW

**Comment by btravouillon** [comment]
Can you wrap the linea to 80 columns ?

File: docs/Userguide_login.rst [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+informations:
+https://sites.google.com/mila.quebec/mila-intranet/it-infrastructure/it-onboarding-training
+
+**IMPORTANT** : Your access to the Cluster is granted based on your status at
+MILA (for students, your status is the same as your main supervisor' status),
+and on the duration of your stay, set during the creation of your account. The
+following have access to the cluster : **Current Students of Core Professors -
+Core Professors - Staff**
+
+
```
Comment (satyaog) [inline docs/Userguide_login.rst:19]: Best to keep the same number of empty line between paragraph for consistency (so 2 instead of 3 in this case)

**Review by satyaog — CHANGES_REQUESTED** [review]

**Comment by satyaog** [comment]
Thanks @gandarioult for your PR! If this is related to #71, you should add the following to the description of this PR:

```

resolves #71
```

This will make GitHub close issue #71 when this PR will be merged.

**Comment by gandarioult** [comment]
Added 'resolves #71' to the description + committed all suggestions

File: docs/Userguide_login.rst [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
+**IMPORTANT** : Your access to the Cluster is granted based on your status at
+Mila (for students, your status is the same as your main supervisor' status),
+and on the duration of your stay, set during the creation of your account. The
+following have access to the cluster : **Current Students of Core Professors -
+Core Professors - Staff**
+
+
 
 SSH Login
```
Comment (satyaog) [inline docs/Userguide_login.rst:21]: Hehe there's still 3 new lines here but that's minor an I approve this PR

```suggestion
Core Professors - Staff**


SSH Login
```

**Review by satyaog — APPROVED** [review]

**Review by achillemyette1 — APPROVED** [review]

**Comment by btravouillon** [comment]
@fosterrath-mila un avis avant merge ?

**Pull request merged**